pkg/server/server_controller_channel_orchestrator.go

```diff
@@ -106,7 +106,10 @@ var _ serverState = (*serverStateUsingChannels)(nil)
 
 // getServer is part of the serverState interface.
 func (s *serverStateUsingChannels) getServer() (orchestratedServer, bool) {
-	return s.server, s.started.Get()
+	if s.started.Get() {
```

Comment:
This change seems incorrect. It was already the case that the callers of this method should not consider the 1st return value if the 2nd was false.

you observing a difference means there are some callers that were not using the API properly. That is the bug we need to find and fix.

I have searched all the current callers of this, and I cannot see any one of them using the API incorrectly. 

Can you share the steps you used to assert this is making a difference?